framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -97,7 +107,9 @@ export const CreatableSelectDropdown = <T extends string = string>({
   menuPlacement = 'bottom',
   ...restProps
 }: CreatableSelectDropdownProps<T>) => {
-  const [ selectedOption, setSelectedOption ] = useState<Option | null>(() => {
+  const [ selectedOption, setSelectedOption ] = useState<Option<
+  T | CreationOptionValue
+  > | null>(() => {
```

Comment:
is it possible to do it in one line?
otherwise let's split it on a separate line after `=` sign:

```
const [ selectedOption, setSelectedOption ] = 
  useState<Option<T | CreationOptionValue> | null>(() => {
```